src/components/molecules/delimited-list/index.tsx

```diff
@@ -0,0 +1,45 @@
+import React from "react"
+import Tooltip from "../../atoms/tooltip"
+
+type DelimitedListProps = {
+  list: Record<string, any>[]
+  delimit?: number
+  property?: string
+}
+
+const DelimitedList: React.FC<DelimitedListProps> = ({
+  list,
+  delimit = 1,
+  property = "name",
+}) => {
+  if (!list.length) {
+    return <></>
+  }
+
+  const ToolTipContent = () => {
+    return (
+      <div className="flex flex-col">
+        {list.slice(delimit).map((listItem) => (
```

Comment:
suggestion: assign to a const and then use. Also, one question, does slicing here impact the component that use this mol?

Reply:
I think as long as the list changes, the slice will follow effect.

Reply:
What i mean is will the slice effect the list of the parent component

Reply:
No, it doesn't. Especially not with the latest commit.